test/runtests.py

```diff
@@ -347,65 +381,60 @@ def timeout_func(timeout_data):
         # check output
         if 'baseline' not in test:
             # output lines must be 'pass' or 'passed' or empty
-            lines = (line.lower() for line in js_output.split('\n'))
-            if any(line != '' and line != 'pass' and line != 'passed'
+            lines = (line.lower() for line in js_output.split(b'\n'))
+            if any(line != b'' and line != b'pass' and line != b'passed'
                     for line in lines):
                 return self._show_failed(**fail_args)
         else:
             baseline = test.get('baseline')
             if baseline:
                 # perform baseline comparison
                 baseline = self._check_file(working_path, baseline)
-                expected_output = None
-                with open(baseline, 'r') as bs_file:
-                    baseline_output = bs_file.read()
-
-                # Cleanup carriage return
-                # todo: remove carriage return at the end of the line
-                #       or better fix ch to output same on all platforms
-                expected_output = re.sub('[\r]+\n', '\n', baseline_output)
-
-                # todo: implement wild cards support
+                with open(baseline, 'rb') as bs_file:
+                    expected_output = bs_file.read().replace(b'\r', b'')
```

Comment:
One reason I changed this is to be consistent with what we do when capturing ch output (`proc.communicate()[0].replace`). Testing CR with baseline is corner case / problematic and we should not do.